config.go

```diff
@@ -213,6 +213,7 @@ type WriterConfig struct {
 	BloomFilters         []BloomFilterColumn
 	Compression          compress.Codec
 	Sorting              SortingConfig
+	SkipPageBounds       []columnPath
```

Comment:
We should use []string here or it wouldn't be possible to initialize the value because columnPath is unexported.

Reply:
updated